src/nodes/NodeConnectionManager.ts

```diff
@@ -1071,7 +1071,6 @@ class NodeConnectionManager {
       this.logger.debug(
         `single connection already existed, cleaning up ${address.host}:${address.port}`,
       );
-      throw Error('haha error go brrrr');
```

Comment:
What...

Reply:
That was my mistake, it was there for testing in a previous PR and wasn't removed.